kclvm/tools/src/util/tests.rs

```diff
@@ -70,8 +72,14 @@ mod test_loader {
                     LoaderKind::JSON,
                     &format!("{}{}", test_case, FILE_EXTENSIONS[0]),
                 );
+
+                #[cfg(not(target_os = "windows"))]
+                let got_data = json_loader.get_data();
+                #[cfg(target_os = "windows")]
+                let got_data = json_loader.get_data().replace("\r\n", "\n");
```

Comment:
Fix the place where the data generates the newline character instead of replacing it here.

Reply:
I think KCL, like Rust, should have used the LF ending no matter what platform it runs on. CRLF is an anachronism that needs to die. So, I adapted the test cases so that they would run on windows without changing the KCL code.